tests/gnrc_dhcpv6_client/tests-as-root/01-run.py

```diff
@@ -50,36 +71,22 @@ def extract_global_address(child):
     return child.match.group("global_addr")
 
 
-def extract_global_addresses(child):
-    """Extract two global addresses and return them as a tuple."""
-    return extract_global_address(child), extract_global_address(child)
+def check_prefix(addr, prefix):
+    return addr.startswith(prefix)
 
 
-def check_ia_na_addr(ia_na_addr):
+def check_ia_na_addr(ia_na_addr, global_pfx):
     """Check if the expected IA_NA address has been assigned"""
-    return IPv6Address(ia_na_addr) in IPv6Network("{}/64".format(IA_NA_ADDRESS_POOL_PREFIX))
+    result = IPv6Address(ia_na_addr) in IPv6Network(f"{IA_NA_ADDRESS_POOL_PREFIX}/64")
+    result &= check_prefix(ia_na_addr, global_pfx)
```

Comment:
Are you sure you want to use bit-wise AND here?

Reply:
Oh, you are right, I somehow thought this was equivalent to `result = result and check_prefix(ia_na_addr, global_pfx)`.